views/your-details.html

```diff
@@ -45,7 +45,7 @@ <h1 class="govuk-heading-xl">{{ i18n.yourDetails.header }}</h1>
                             {{ type }} {{i18n.yourDetails.emailLabel}}
                         </th>
                         <td class="govuk-table__cell {{ hideBottomBorder }}">
-                            {{ value.email }}
+                            {{ value.email if value.email else i18n.yourDetails.notProvided }}
```

Comment:
Here it will print 'undefined'/null then "Not provided"

Reply:
Thanks, Sunil. Is it displaying undefined/null when you run it? It seems to be working fine for me:
![Screenshot 2020-03-04 at 11 54 17](https://user-images.githubusercontent.com/14298372/75877487-c6242780-5e0f-11ea-8341-063890dbc61b.png)